### PR TITLE
feat(nvmf/nqn): allow using custom nvme nqn prefix

### PR DESCRIPTION
**Pull request by tiagolobocastro**

### CODE REVIEW

**Comment by tiagolobocastro** [comment]
bors try

**Comment by tiagolobocastro** [comment]
bors try

**Comment by tiagolobocastro** [comment]
testing with other nqn:
bors try

**Comment by tiagolobocastro** [comment]
bors try

File: io-engine/src/constants.rs [hunk 1]
```diff
@@ -2,10 +2,10 @@
 pub const NEXUS_CAS_DRIVER: &str = "Nexus_CAS_Driver";
 
 /// Data plane's NVMe controller model ID.
-pub const NVME_CONTROLLER_MODEL_ID: &str = "Mayastor NVMe controller";
+pub const NVME_CONTROLLER_MODEL_ID: &str = "Other NVMe controller";
```
Comment (Abhinandan-Purkait) [inline io-engine/src/constants.rs]: Other --> Mayastor ?

File: io-engine/src/constants.rs [hunk 2]
```diff
@@ -2,10 +2,10 @@
...
 
 /// NVMe NQN prefix.
-pub const NVME_NQN_PREFIX: &str = "nqn.2019-05.io.openebs";
+pub const NVME_NQN_PREFIX: &str = "nqn.2019-05.com.name";
```
Comment (Abhinandan-Purkait) [inline io-engine/src/constants.rs]: com.name --> io.openebs?

File: test/python/common/constants.py [hunk 3]
```diff
@@ -0,0 +1 @@
+nvme_nqn_prefix = "nqn.2019-05.com.name"
```
Comment (Abhinandan-Purkait) [inline test/python/common/constants.py]: com.name --> openebs.io?

File: test/grpc/test_common.js [hunk 4]
```diff
@@ -19,6 +19,9 @@ const GRPC_PORT = 10124;
 const CSI_ENDPOINT = '/tmp/mayastor_csi_test.sock';
 const CSI_ID = 'test-node-id';
 const LOCALHOST = '127.0.0.1';
+const NVME_MODEL_ID = 'Other NVMe controller';
```
Comment (Abhinandan-Purkait) [inline test/grpc/test_common.js]: Other -> Mayastor

com.name --> openebs.io below?

Comment (Abhinandan-Purkait) [reply test/grpc/test_common.js]: Maybe the regex as well below?

**Review by Abhinandan-Purkait — APPROVED** [review]

File: io-engine/src/constants.rs (hunk 1 above)
Comment (tiagolobocastro) [reply io-engine/src/constants.rs]: hum... these were just here for testing...

**Review by dsharma-dc — APPROVED** [review]

**Comment by tiagolobocastro** [comment]
bors merge

**Pull request merged**